src/main/java/core/basesyntax/service/data/DataToOperationsFormatter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.data;
+
+import core.basesyntax.model.Operation;
+import java.util.List;
+
+public interface DataToOperationsFormatter {
+    List<Operation> formatData(List<String> data);
+}
```

Comment:
Overall this is a good solution. In many libraries, it could be like
```suggestion
public interface DataParser<T, U> {
    List<T> parse(List<U> data);
}
```
and implementation could be like
```public class CsvOperationParser implements DataParser<Operation, String>```.
You don't need to refactor this code, this is mostly a hint for the future.

Reply:
refactored and renamed